### DISCUSSION TITLE
Spotify --> Librespot --> Snapcast (Shows Connected and Playing but Stream Idle)

### DISCUSSION THREAD

**Discussion by cpttater** [snapcast/snapcast#1146] [Q&A]
Hello, 

I have been using this setup for a while and it has been working great. I recently tried to add my Sons account but when I select one of my librespot Spotify Connect Streams it will connect and show I am listening on the Spotify APP but the stream will remain idle. 

When I restart the snapcast process on my Ubuntu host the connection gets severed and the Spotify APP will also show that it disconnected. 

Any ideas? 
https://imgur.com/jCrL8kX

**Comment by kingosticks** [comment]
First step is to make things simpler. Can you run librespot standalone and see if there's anything in the log?
